guides/micronaut-cloud-trace-google/micronaut-cloud-trace-google.adoc

```diff
@@ -0,0 +1,100 @@
+external:micronaut-cloud-trace-base/start.adoc[]
+* A https://cloud.google.com/gcp/[Google Cloud Platform (GCP)] account and a GCP project.
+
+common:costs-gcloud.adoc[]
+
+common:gcp.adoc[]
+
+external:micronaut-cloud-trace-base/create-app.adoc[]
+
+=== Tracer Configuration
+
+Using Micronaut CLI or Launch to create your application, the OpenTelemetry exporter will be added to your `application.yml` configurations.
+
+resource:application.yml[tag=otel-trace]
+
+== Run the Application
+
+The application can be deployed to Google Cloud Run, Cloud Function, Cloud Compute or App Engine.
+
+Traces can be sent to Google Cloud Trace outside the Google Cloud.
+
+This allows us to run the application locally and see the traces in Google Cloud Trace.
+
+common:runapp-instructions.adoc[]
+
+[NOTE]
```

Comment:
Should this note include a comment like, "This is normal and will not impact your application performance" or "You can disregard this error message" or "If you get this error message, shut down your machine immediately and perform a standard exorcism ritual before turning it back on"?

Reply:
Fixed in https://github.com/micronaut-projects/micronaut-guides/pull/964